iam/scopes/scopes.go

```diff
@@ -19,7 +19,7 @@ const (
 	TypeTargetGroup Type = "targetGroup"
 	TypeAccount     Type = "account"
 
-	ResourceDeployments  Resource = "deployments"
+	ResourceDeployment   Resource = "deployment"
```

Comment:
felt like this should be consistent. I checked in prod and no one has created a `deployments` resourced grant, so this is OK to change.